eth/eventhandler/event_handler_test.go

```diff
@@ -203,6 +205,23 @@ func TestHandleBlockEventsStream(t *testing.T) {
 	// Receive event, unmarshall, parse, check parse event is not nil or with an error,
 	// public key is correct, owner is correct, operator ids are correct, shares are correct
 	t.Run("test ValidatorAdded event handle", func(t *testing.T) {
+		// using some random private key for testing
+		operatorEthKey, err := crypto.HexToECDSA("42e14d227125f411d6d3285bb4a2e07c2dba2e210bd2f3f4e2a36633bd61bfe6")
+		require.NoError(t, err)
+		operatorEthAdr := crypto.PubkeyToAddress(operatorEthKey.PublicKey)
+
+		eh.operatorData.SetOperatorData(&registrystorage.OperatorData{
+			PublicKey:    ops[0].pub,
+			ID:           ops[0].id,
+			OwnerAddress: operatorEthAdr,
+		})
+		_, err = eh.nodeStorage.SetupPrivateKey(base64.StdEncoding.EncodeToString(ops[0].priv))
```

Comment:
I don't get why you set this twice, once here and once in `setupEventHandler`

Reply:
Fixed